features/journalist_can_edit_and_delete_articles.feature

```diff
@@ -42,11 +42,3 @@ Feature: Journalist can edit and delete articles
         And I visit the "Voted best mead recipe" page
         And I click "Edit"
         Then I should see "You can only edit your own articles."
-
-    @javascript
```

Comment:
Any reason this was removed?

Reply:
it's not removed. just moved it up a bit